clients/drcachesim/scheduler/scheduler.h

```diff
@@ -916,7 +931,9 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
         /**
          * Returns a unique identifier for the current output stream.  For
          * #MAP_TO_RECORDED_OUTPUT, the identifier is the as-traced cpuid mapped to this
-         * output.  For dynamic schedules, the identifier is the output stream ordinal.
+         * output.  For dynamic schedules, the identifier is the output stream ordinal,
+         * except for #OFFLINE_FILE_TYPE_CORE_SHARDED inputs where the identifier
```

Comment:
For OFFLINE_FILE_TYPE_CORE_SHARDED we have input_stream_ordinal == output_stream_ordinal right? If so, do we need this clarification?

Reply:
Actually if #workers < #shards they are not the same as multiple inputs will run on a single output.